### PR TITLE
Updates to CC teacher dashboard

### PR DESCRIPTION
**Pull request by nathanstitt**
- [x] Split apart into separate files
- [x] change "Class Performance" to "Class Dashboard"
- [x] sort chapters descending and sections ascending
- [x] align the chapter/sections (and make them black, not red)
- [x] add (i) bubbles
- [x] make every other row gray
- [x] try to move "detailed scores" next to the period tabs
- [x] have some boilerplate text for when a class is empty
- [x] only show the chapter_section when >= 10% of students complete it
- [x] move PDF/Webview links up to course title (empty if not CC) - depends on https://github.com/openstax/tutor-server/pull/870
- [ ] boilerplate text when this is empty
- [ ] boilerplate text when there are no periods

### CODE REVIEW

**Comment by Fredasaurus** [comment]
pics or it didn't happen.

File: resources/styles/components/cc-dashboard/index.less [hunk 1]
```diff
@@ -1,66 +1,98 @@
-.dashboard-header-links {
-  .link {
-    color: @tutor-tertiary-light;
-    text-transform: capitalize;
-    margin-left: 10px;
-    &.btn {
-      padding: 0;
+cc-dashboard {
+  .column-legend {
+    span {
```
Comment (philschatz) [inline resources/styles/components/cc-dashboard/index.less]: Is this a `react-bootstrap` class? If so, should the selector contain `>` so the styling doesn't bleed?

Comment (nathanstitt) [inline resources/styles/components/cc-dashboard/index.less]: no it's just a span.. actually i'll just target children there I think

File: src/components/cc-dashboard/index.cjsx [hunk 2]
```diff
@@ -1,184 +1,12 @@
...
-
-  render: ->
-    courseId = @props.id
-    periods = CCDashboardStore.getPeriods(courseId)
-    periodChapters = CCDashboardStore.getPeriodChapters(courseId, @state.activePeriodId)
-
-    if periodChapters?.length
-      chapters = _.map periodChapters, @renderChapters
-    else
-      chapters = <div>There are no completed concept coach tasks for this period.</div>
```
Comment (philschatz) [inline src/components/cc-dashboard/index.cjsx]: Why was this message removed?

Comment (nathanstitt) [inline src/components/cc-dashboard/index.cjsx]: It's being set in the chapter,  I'd just missed removing it from that

**Pull request merged**